src/components/Shell/_mixins.scss

```diff
@@ -101,24 +101,16 @@
   }
 
   &__skip-to-content {
-    display: flex;
     position: relative;
-    height: 0;
+    display: flex;
     justify-content: center;
 
-    &__link {
-      @include hidden;
-
+    &__link.#{$prefix}--skip-to-content:focus {
+      position: relative;
+      border-color: $focus;
+      background-color: $interactive-01;
       margin-top: $header__spacing__padding;
-      margin-left: -$toolbar__sizing__width;
-      transition: unset;
-      z-index: z($layer: floating);
```

Comment:
Something worth noting here -- `floating` is 10,000, and Carbon's `SkipToContent` is set to 9,999 by default.